pyproject.toml

```diff
@@ -41,22 +33,81 @@ classifiers = [
     "Topic :: System :: Distributed Computing",
 ]
 
-[project.urls]
-Homepage = "https://github.com/locustio/locust"
-Documentation = "https://docs.locust.io/"
-Code = "https://github.com/locustio/locust"
-"Help/Questions" = "https://stackoverflow.com/questions/tagged/locust"
-"Issue tracker" = "https://github.com/locustio/locust/issues"
+packages = [{ include = "locust" }]
+include = [
+    "locust/webui/dist/**/*",
+    "locust/_version.py",
+    { path = "poetry.lock", format = [
+        "sdist",
+    ] },
+]
+exclude = [
+    "locust/webui/public",
+    "locust/webui/src",
+    "locust/webui/*.*",
+    "locust/test",
+]
 
+[tool.poetry.build]
+script = "build.py"
+generate-setup-file = false
 
-[project.readme]
-file = "README.md"
-content-type = "text/markdown"
+[tool.poetry.urls]
+"Help/Questions" = "https://stackoverflow.com/questions/tagged/locust"
```

Comment:
Would Slack be a more useful link than Stackoverflow for questions? I don't know if any Locust maintainers are active on Stackoverflow

Reply:
I am :) it is linked to the stack-overflow-rss channel on slack. But tbh, I think we should link people to the github discussions forum now.

Reply:
So https://github.com/orgs/locustio/discussions/categories/q-a should be the URL in this case

Reply:
Updated this to add them all, pretty sure the number/title of the URLs are arbitrary - we can drop SO and Slack links if you want as well 👍